src/uwtools/set_namelist_ingest.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+
+'''
+This utility demonstrates the ingest functions to update a Fortran namelist file
+using the f90nml package while using a Jinja2 templated input file that is
+filled in from the user environment
+'''
+
+import os
+import sys
+import argparse
+
+from jinja2 import Environment, FileSystemLoader, meta
+import f90nml
+
+def parse_args(argv):
+
+    '''
+    Function maintains the arguments accepted by this script. Please see
+    Python's argparse documentation for more information about settings of each
+    argument.
+    '''
+
+    parser = argparse.ArgumentParser(
+       description='Update a Fortran namelist with user-defined settings.'
+    )
+    # output file (must be fully qualified path) true to
+    # spec f90 name list file from python lib f90nml
+    parser.add_argument('-o', '--outfile',
+                       help='Full path to output file. This is a \
+                       namelist by default.',
+                       required=False,
+                       )
+    # Jinja2 user input file as a template to moc f90 name list file
+    parser.add_argument('-i', '--input_nml',
+                       required=True,
+                       help='Path to a templated user namelist.',
+                       )
+    # switch for printing to stdout the f90 namelist output file
+    parser.add_argument('-d', '--dry_run',
+                       action='store_true',
+                       help='If provided, suppress all output.',
+                       )
+    # switch for printing to stdout the required values needed to filled in by the template
+    parser.add_argument('-v', '--values_needed',
```

Comment:
The `-v` flag is most often associated with a verbosity argument, and it would be nice to stay consistent with an unwritten standard in linux utilities. I suggest removing the short flag `-v` and just leaving the long one `--values_needed` as it is descriptive of the type of output a user would want.

Reply:
Agreed! Good idea to just drop the short switch.